client/src/pages/dashboard/Dashboard.svelte

```diff
@@ -74,6 +74,8 @@
 {/await}
 
 <style>
+    @import url('../vars.css');
+

```

Comment:
As mentioned earlier, this shouldn't be necessary anymore since this is implied by `global.css`.
```suggestion
```